JZAndroid/app/build.gradle

```diff
@@ -199,8 +210,10 @@ dependencies {
 
     api "org.apache.commons:commons-lang3:3.6"
 
+    implementation 'com.github.kenglxn.QRGen:android:2.6.0'
+    implementation 'com.opencsv:opencsv:3.9'
 
-    testImplementation 'junit:junit:4.12'
```

Comment:
Dere må ikke ta bort JUnit fra enhetstestene. Den er nødvendig både for enhetstester, og for UI-tester. Ellers kan dere eventuelt fjerne eksempeltesten siden det uansett ikke er noen reelle enhetstester.